src/main/java/Entities/Player.java

```diff
@@ -179,28 +166,6 @@ private HashMap<String, Integer> createSetMap() {
         return sets;
     }
```

Comment:
this code should be removed due to the fact that the original method is moved to the playerlogic class